Content.Client/GameObjects/Components/Configuration/ConfigurationMenu.cs

```diff
@@ -26,7 +26,7 @@ public ConfigurationMenu(ConfigurationBoundUserInterface owner)
 
             _inputs = new List<(string name, LineEdit input)>();
 
-            Title = Loc.GetString("Device Configuration");
+            Title = Loc.GetString("configuration-device");
```

Comment:
I like the `-title` suffix you had for some of the other menus, it'd be nice if that was used consistently!